code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -60,7 +60,6 @@
 
 	var/alarms = list("Motion"=list(), "Fire"=list(), "Atmosphere"=list(), "Power"=list(), "Camera"=list(), "Burglar"=list())
 
-	var/speed = 0 /// VTEC speed boost.
```

Comment:
did you make sure this didn't break anything?

Reply:
I think it shouldnt